fe/fe-core/src/main/java/com/starrocks/master/ReportHandler.java

```diff
@@ -110,6 +120,10 @@ public Long getValue() {
             }
         };
         MetricRepo.addMetric(gaugeQueueSize);
+        pendingTaskMap.put(ReportType.TABLET_REPORT, Maps.newHashMap());
+        pendingTaskMap.put(ReportType.DISK_REPORT, Maps.newHashMap());
+        pendingTaskMap.put(ReportType.TASK_REPORT, Maps.newHashMap());
```

Comment:
I think one be can have multiple `TASK REPORT` at the same time. If only one task is kept I think many task reports will be ignored.